packages/hoprd/src/index.ts

```diff
@@ -197,6 +200,18 @@ const argv = yargs(process.argv.slice(2))
     default: false,
     hidden: true
   })
+  .option('testHeartbeatInterval', {
```

Comment:
Why use a test prefix? This seems like a valid configuration option for any node.

Reply:
changed